src/Build/PostBuild.cmd

```diff
@@ -16,10 +17,11 @@ copy /y "%SolutionDir%SDK\OrleansConfiguration.xml" "%TargetDir%"
 copy /y "%SolutionDir%SDK\ClientConfiguration.xml" "%TargetDir%"
 
 if "%BuildOrleansNuGet%" == "" (
+    set BuildOrleansNuGet=true
     if "%Configuration%" == "Release" (
-        set BuildOrleansNuGet=true
+        set BuildOrleansNuGetPrerelease=false
     ) else (
-        set BuildOrleansNuGet=false
+        set BuildOrleansNuGetPrerelease=true
```

Comment:
similarly to BuildOrleansNuGet, I think it's probably better to only set BuildOrleansNuGetPrerelease if it's not already set.